minishift_sanity.py

```diff
@@ -92,90 +92,164 @@ def test_ms_start(self):
         else:
             self.fail("Minishift start failed")
             provision_path = None
+    """
     
-    """        
+    def test_ms_setup_cdk(self):
+        '''
+        Testing if setup-cdk command is able to setup CDK properly:
+        RHEL iso and OCP binary are in place
+        '''
+        cmd = "minishift setup-cdk"
+        self.log.info("Running command: " + cmd)
+        child = pexpect.spawn(cmd)
+        index = child.expect(["CDK 3 setup complete.", pexpect.EOF, pexpect.TIMEOUT], timeout=30)
+        if not index==0:
+            self.fail("CDK setup failed")
+        self.log.info("Checking if rhel.iso exists")
+        exists = os.path.isfile( os.environ['HOME'] + "/.minishift/cache/iso/minishift-rhel.iso" )
+        if not exists:
+            self.fail("minishift-rhel.iso is not present")
+        self.log.info("Checking if oc binary exists")
+        exists = os.path.isfile( os.environ['HOME'] + "/.minishift/cache/oc/v3.4.0/oc" )
+        if not exists:
+            self.fail("oc binary is not present")
+        self.log.info("Checking if config.json exists")
+        exists = os.path.isfile( os.environ['HOME'] + "/.minishift/config/config.json" )
+        if not exists:
+            self.fail("config.json binary is not present")
+        self.log.info("CDK Installation successful")
+
     def test_ms_start(self):
-        if iso_url == "":
-            cmd = "minishift start --vm-driver " +self.Hypervisor_Provider
-        else:
-            cmd = "minishift start --vm-driver " +self.Hypervisor_Provider +" --iso-url " +self.iso_url
-        self.log.info("Starting minishift...")
+	if self.iso_url:
```

Comment:
Python handles "" in if conditional as false result.
```
if "":
    print "it is true"
else:
    print "not true"
>> not true
```
I am using it like this quite often but 1. I am not sure if it is ok with PEP-8 and 2. not sure if it is widely known, so I can change it to self.iso_url == "" which is maybe more readable, WDYT?